RELEASE_PROCESS.md

```diff
@@ -0,0 +1,58 @@
+# GraphiQL Monorepo Release Process
+
+This document should guide you through doing a release of GraphiQL. Much of
+the release process is automated, so just follow this process through and
+everything should go smoothly!
+
+Note: we run all `lerna` commands with `npx lerna` to ensure that we're using
```

Comment:
this can also be accomplished with `yarn exec`

Reply:
`npx` runs the local copy if there is one; it's shorter than but equivalent to `./node_modules/.bin/lerna` assuming that the modules are installed.